### DISCUSSION TITLE
How do you control context

### DISCUSSION THREAD

**Discussion by Liu-Angelo** [acheong08/ChatGPT#747] [Q&A]
revChatGPT.V1 import Chatbot  
  
  conversation_id=chatbot.config.get("conversation"),
  parent_id=chatbot.config.get("parent_id"),
  
  example：
 ------------------------------------------
  from revChatGPT.V1 import Chatbot

chatbot = Chatbot(config={
  "email": "<your email>",
  "password": "your password"
})

for data in chatbot.ask(
  prompt,
  conversation_id=chatbot.config.get("conversation"),
  parent_id=chatbot.config.get("parent_id"),
):
  print(data["message"], end="", flush = True)
print()

**Comment by acheong08** [comment]
Context of? If you mean conversations, check  https://github.com/acheong08/ChatGPT/wiki/V1 for reference

**Reply by x281737830** [reply]
What does parent_id mean? There's no explation for the funtion an parameters in https://github.com/acheong08/ChatGPT/wiki/V1

**Reply by acheong08** [reply]
`parent_id` is the message id of the previous response. The library fetches the latest one automatically if you only provide a `conversation_id`
